### PR TITLE
FLUID-6677: add Infusion 3.0 release notes

### PR DESCRIPTION
**Pull request by jobara**
https://issues.fluidproject.org/browse/FLUID-6677

### CODE REVIEW

**Comment by jobara** [comment]
@amb26 could you please review this PR to update the release notes for the Infusion 3.0 release?

File: ReleaseNotes.md [hunk 1]
```diff
@@ -6,35 +6,73 @@
...
+  * Minified distributions:
+    * infusion-all.js
+    * infusion-all-no-jquery.js
+    * infusion-framework.js
+    * infusion-framework-no-jquery.js
+    * infusion-uio.js
+    * infusion-uio-no-jquery.js
+* Framework
+  * Added model transformations for converting between:
+    * Boolean values and Strings
```
Comment (amb26) [inline ReleaseNotes.md:21]: List what the relevant grades are for these three

File: ReleaseNotes.md [hunk 2]
```diff
@@ -6,35 +6,73 @@
...
+    * Date/Time and Strings
+    * JSON Objects and Strings
+  * Updated model transformations:
+    * Number to String transformation supports specifyhing decimal precission.
```
Comment (amb26) [inline ReleaseNotes.md:25]: Typo precision

File: ReleaseNotes.md [hunk 3]
```diff
@@ -6,35 +6,73 @@
...
+    * Round tranformation can round to an integer or decimal value
+  * `fluid.stringTemplate` updated to support nested objects for interpolate values
```
Comment (amb26) [inline ReleaseNotes.md:27]: Typo interpolating

File: ReleaseNotes.md [hunk 4]
```diff
@@ -6,35 +6,73 @@
...
-* Preference framework
-  * Collapsed inputsLarger and emphasizeLinks preferences into enhanceInputs
+  * Added localized message bundles for Farsi, French, Portugese, and Spanish.
+  * New preferences:
+    * Letter spacing
+    * Syllabification preference
+    * Text-to-speech preference using the Orator component
+    * Word spacing preference
+* Orator
+  * A self voicing widget with play/pause, text highlighting, selection reading.
```
Comment (amb26) [inline ReleaseNotes.md:43]: The problems with the Chrome TTS implementation that prevent this from currently being usable are probably serious enough to warrant mentioning here

**Comment by jobara** [comment]
@amb26 thanks for the feedback. Ready for more review.

**Pull request merged**